inference-engine/src/cldnn_engine/cldnn_engine.cpp

```diff
@@ -617,7 +617,7 @@ IExecutableNetworkInternal::Ptr clDNNEngine::LoadExeNetworkImpl(const InferenceE
 }
 
 IExecutableNetworkInternal::Ptr clDNNEngine::LoadExeNetworkImpl(const InferenceEngine::CNNNetwork &network,
-                                                                const RemoteContext::Ptr &context,
+                                                                const IRemoteContext::Ptr &context,
```

Comment:
I propose to minimize impact on old public API

Reply:
It is not a public API. It is an internal Plugin API.